assets/src/application/ui/input/Button/style.scss

```diff
@@ -46,7 +46,7 @@
 		font-size: var(--ee-font-size-default);
 		font-weight: bold;
 		line-height: var(--ee-font-size-default);
-		margin: 0 0 var(--ee-margin-smaller);
+		// margin: 0 0 var(--ee-margin-smaller); Setting margins on these states is a bit confusing, needs discussion.
```

Comment:
this margin was causing this issue:

![image](https://user-images.githubusercontent.com/1477453/79993916-f4dc8700-84bd-11ea-9d5d-1180a9e6eeb2.png)

Reply:
I don't quite understand the reasoning behind this, it might be trying to override other styles, but not sure which one and not sure if the is still actual.